task60_PageObject/src/test/java/tut/by/pages/HomePage.java

```diff
@@ -0,0 +1,34 @@
+package tut.by.pages;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+
+import static org.testng.Assert.assertTrue;
+
+public class HomePage  extends BasePage {
+    private By LOGIN_WINDOW_BUTTON = By.className("enter");
+    private By USER_NAME = By.className("uname");
+    private By LOGOUT_BUTTON = By.className("auth__reg");
+
+    public HomePage(WebDriver driver){
+       super(driver);
+       assertTrue(isDisplayed(LOGIN_WINDOW_BUTTON));
+    }
+
+    public void openLoginWindow(){
```

Comment:
According to classic page object: all public methods must return current page/new page or some value. Update all public void methods

Reply:
Done, I've updated all public void methods except methods in BasePage class.